src/sidekiq/server/cli.cr

```diff
@@ -5,15 +5,15 @@ require "../server"
 
 module Sidekiq
   class CLI
-    getter logger : ::Logger
+    getter logger : ::Log = ::Log.for("sidekiq")
```

Comment:
I think it should be `Sidekiq::Logger.build("sidekiq")`. `::Log.for("sidekiq")` bypasses some setup steps in `Sidekiq::Logger.build`. The result is an empty output when starting the worker.

Reply:
btw, awesome work :+1: